SpeechResponder/ScriptResolver.cs

```diff
@@ -1186,9 +1186,13 @@ CommodityMarketQuote CommodityDetails(string commodityLocalizedName, Station sta
 
                 if (curr?.x != null && dest?.x != null)
                 {
-                    result = (decimal)Math.Round(Math.Sqrt(square((double)(curr.x - dest.x))
-                                + square((double)(curr.y ?? 0 - dest.y ?? 0))
-                                + square((double)(curr.z ?? 0 - dest.z ?? 0))), 2);
+                    result = (decimal)Math.Round(
+                        Math.Sqrt(
```

Comment:
I would hoist this to a local variable, perhaps called `distance`. 
Zero cost and increases readability.

Reply:
Done.